testsuite/MDAnalysisTests/test_streamio.py

```diff
@@ -19,7 +19,7 @@
 import MDAnalysis
 import MDAnalysis.lib.util as util
 import MDAnalysis.tests.datafiles as datafiles
-from MDAnalysisTests.test_coordinates import RefAdKSmall
+from MDAnalysisTests.coordinates.reference import RefAdKSmall
```

Comment:
Does anyone have an idea why this fails on travis? I can't reproduce the error on my machine.